tools/snp.sh

```diff
@@ -199,6 +203,11 @@ install_rust() {
 
   # Install rust
   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs -sSf | sh -s -- -y
+  #Beeded to build Cococnut-SVSM
+  if "$SVSM"; then
+    rustup target add x86_64-unknown-linux-musl
+    cargo install cbindgen
+  fi
```

Comment:
```suggestion
  # Needed to build Cococnut-SVSM
  if "$SVSM"; then
    rustup target add x86_64-unknown-linux-musl
    cargo install cbindgen
  fi
```

@ryansavino it may make sense to just build the musl for the entire project (fixing the same issue with backward compatibility with older distros, etc. Thoughts?